dotcom-rendering/src/web/server/render.ts

```diff
@@ -48,7 +48,7 @@ export const renderArticleJson = (
 		};
 
 		res.status(200).send(resp);
-	} catch (e) {
+	} catch (e: any) {
```

Comment:
Sorry, completely unrelated to the substance of your change, but I'm curious as to why you've added the explicit `any` here?

Reply:
I just added that to get rid of the red squiggly line I had in my code editor! Not proposing we move forward with this (but some form of typing needs to be added)

Reply:
:-) Fair enough.

`unknown` seems most accurate since things can throw anything.  Although in this example I think we'd only have a problem if something throws `undefined` or `null`.